src/backend/distributed/sql/udfs/citus_finish_pg_upgrade/11.0-1.sql

```diff
@@ -104,7 +104,7 @@ BEGIN
     --
     -- register triggers
     --
-    FOR table_name IN SELECT logicalrelid FROM pg_catalog.pg_dist_partition
+    FOR table_name IN SELECT logicalrelid FROM pg_catalog.pg_dist_partition JOIN pg_class ON (logicalrelid = oid) WHERE relkind <> 'f'
```

Comment:
If we do this, how are already Citus 11 users would get this?

We should need something like: https://github.com/citusdata/citus/pull/6037/files

@hanefi do you agree?

Reply:
Yes, that is correct. I will start reviewing and testing if the updated migration versions are fine now.